asab/application.py

```diff
@@ -128,7 +128,11 @@ def handler(type):
 		self.TaskService = TaskService(self)
 
 		# Setup ASAB API
-		if len(Config['asab:web']["listen"]) > 0:
+		if (
+			Config.has_option("web", "listen") and len(Config.get("web", "listen")) > 0
```

Comment:
`(Config.has_option("web", "listen") and len(Config.get("web", "listen")) > 0) or ...` ...

Be explicit, that's much more future-proof